packages/data-transformation/fhir-to-csv/src/parseFhir/parseFhir.py

```
@@ -0,0 +1,341 @@
+import json
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)